src/core/lib/channel/context.h

```diff
@@ -45,6 +45,10 @@ typedef enum {
   /// Holds a pointer to ServiceConfigCallData associated with this call.
   GRPC_CONTEXT_SERVICE_CONFIG_CALL_DATA,
 
+  /// Holds a pointer to BackendMetricProvider associated with this call on
+  /// the server.
+  GRPC_CONTEXT_BACKEND_METRICS,
```

Comment:
Done. But should we instead name it GRPC_CONTEXT_BACKEND_METRIC_FILTER? In case we later want to add a second member and change it to a holder struct.